x-pack/plugins/cases/public/components/add_comment/index.tsx

```diff
@@ -102,16 +111,23 @@ export const AddComment = React.memo(
                   dataTestSubj: 'add-comment',
                   placeholder: i18n.ADD_COMMENT_HELP_TEXT,
                   bottomRightContent: (
-                    <EuiButton
-                      data-test-subj="submit-comment"
-                      iconType="plusInCircle"
-                      isDisabled={isLoading}
-                      isLoading={isLoading}
-                      onClick={onSubmit}
-                      size="s"
-                    >
-                      {i18n.ADD_COMMENT}
-                    </EuiButton>
+                    <EuiFlexGroup gutterSize="s" alignItems="flexEnd" responsive={false} wrap>
```

Comment:
I think it is a good opportunity to move these components (inside `bottomRightContent`) outside of the `AddComment` component to `UserActionTree`. This will help to isolate the responsibility of the form field.

Reply:
Let's leave it for another PR.